### PR TITLE
Update USDT address

### PR DESCRIPTION
**Pull request by qiwu7**
Fixes # .

### Motivation
Update the obsolete USDT address

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func getSupportedTokens(network string) map[string]bool {
 			"0x4200000000000000000000000000000000000042": true, // OP
 			"0xda10009cbd5d07dd0cecc66161fc93d7c9000da1": true, // DAI
 			"0x2e5ed97596a8368eb9e44b1f3f25b2e813845303": true, // SNX
-			"0x7f5c764cbc14f9669b88837ca1490cca17c31607": true, // USDT - obsolete
+			"0x853eb4bA5D0Ba2B77a0A5329Fd2110d5CE149ECE": true, // USDT
```
Comment (dneilroth) [inline cmd/run.go:154]: looks like all other addresses are lowercased

File: .gitignore [hunk 2]
```diff
@@ -2,3 +2,4 @@ rosetta-ethereum
 ethereum-data
 cli-data
 .env
+.idea/
```
Comment (dneilroth) [inline .gitignore:5]: what is this one?

File: cmd/run.go (hunk 1 above)
Comment (qiwu7) [reply cmd/run.go:154]: hmm

File: .gitignore (hunk 2 above)
Comment (qiwu7) [reply .gitignore:5]: This is for IntelliJ

File: cmd/run.go (hunk 1 above)
Comment (qiwu7) [reply cmd/run.go]: updated with lowercase

**Review by dneilroth — APPROVED** [review]

**Pull request merged**